bizdemo/easy_note/cmd/api/main.go

```diff
@@ -66,6 +83,15 @@ func main() {
 		TimeFunc:      time.Now,
 	})
 
+	r.Use(recovery.Recovery(recovery.WithRecoveryHandler(
+		func(ctx context.Context, c *app.RequestContext, err interface{}, stack []byte) {
+			hlog.SystemLogger().CtxErrorf(ctx, "[Recovery] err=%v\nstack=%s", err, stack)
+			c.JSON(http.StatusInternalServerError, map[string]interface{}{
+				"code":    errno.ServerPanicErrCode,
+				"message": err.(string),
```

Comment:
i think it should contains stack and err.

Reply:
i got it, thx